src/llama.cpp

```diff
@@ -19948,20 +19958,12 @@ int32_t llama_model_desc(const struct llama_model * model, char * buf, size_t bu
             llama_model_ftype_name(model->ftype).c_str());
 }
 
-uint64_t llama_model_size(const struct llama_model * model) {
-    uint64_t size = 0;
-    for (const auto & it : model->tensors_by_name) {
-        size += ggml_nbytes(it.second);
-    }
-    return size;
+uint64_t llama_model_size(const struct llama_model *model) {
+    return model->n_bytes;
```

Comment:
I think replacing these loops with an implicit cast is fine, but if you don't mind changing the api it could be

```cpp
size_t llama_model_size(const struct llama_model *model) { } 
int64_t llama_model_n_params(const struct llama_model *model) { }
```